en/frames_autogyro/ThunderFly_AutoG2.md

```diff
@@ -0,0 +1,121 @@
+
+Rotor axis, made of M3x50 high tensile strength screw, goes through the third part. Bearings used are 623 2Z C3 SKF. At the end of this part, there are ball rods attached via M2.5 screws to servos positioned in the bottom part of the pylon. It is preferable to exchange these original servos for better quality ones as they are weak and in the original construction they help each other.
+
+![Rotorhead](../../assets/airframes/autogyro/auto-g2/modif_rh.png)
+
+### Two-blade rotor
+Durafly Auto-G2 autogyro has originally a three-blade rotor. Our modification Auto-G2 uses a two-blade rotor. The reasons are reduced vibrations and easier construction. Printed central parts are designed to be used both with chinese Durafly blades or 3D printed blades.
+
+Rotor’s central part consists of several components which have following roles:
+  * They enable blade flapping
+  * They have deformation zones that break upon impact with ground. Thanks to this, the rotor can usually be repaired quickly by replacing only one component.    
+  * Easy setup of blades angle-of-attack.
+
+#### HobbyKing rotor blades
+
+It is possible to use a printed central part of the rotor with the original blades. These blades can be bought on [HobbyKing](https://hobbyking.com/en_us/duraflytm-auto-g-gyrocopter-821mm-replacement-main-blade-1pcs-bag.html). Hobbyking blades differ in the position of center of gravity and it is therefore necessary to balance them properly.
+
+#### 3D printed rotor blades
+Alternatively, it is possible to print blades. They are still under development, but preliminary tests show they are of better quality mostly thanks to their precise shape and absence of longitudinal grooves. However, some of the production processes still need to be tuned.
+
+![Blades assembly](../../assets/airframes/autogyro/auto-g2/modif_blade.png)
+
+#### Balancing
+
+Proper blades balance is very important to minimize vibrations. Blades have to be balanced in such a way, that the center of gravity is located in the middle of the rotor axis.
+
+Printed blades are balanced in the production process and there is no need to further balance them.
+
+### Release device
+
+If you want to launch an autogyro using a winch or if you want to launch it by towing you need to print a release device. It is a small box equipped with servo which pulls out the pin and releases the rope.
+
+The whole part is glued, using a hot-melt adhesive, under the engine on the bottom part of an autogyro’s body. If an autogyro is towed by a rope its engine must not be turned on. It can be handled, for example, by nulling the engine’s output in the transmitter if the release device switch is closed.
+
+![Release device](../../assets/airframes/autogyro/auto-g2/modif_release.png)
+
+## Parts List
+
+### Electronic
+ * Autopilot
+ * GPS
+ * Airspeed sensor
+ * Stronger servos as a substitution for the original ones (optional)
+ * Additional servo for release device (optional)

```

Comment:
What is the release device? Is there a link for this?

Reply:
Our autogyro can be launched on a rope (from winch or towed behind car). The release device is to detaching this rope from the drone.

Reply:
release device is described in own chapter of this text.